ddtrace/_monkey.py

```diff
@@ -48,6 +48,7 @@
     "grpc": True,
     "httpx": True,
     "kafka": True,
+    "langgraph": False,
```

Comment:
defaulting this to `False`, as it's an experimental integration and we want it gated behind explicit enablement (via env var or direct `patch`)

Reply:
I'd rather we omit it from being in `patch_all()` and use `_DD_TRACE_LANGGRAPH_ENABLED` just to be extra clear that it's experimental

Reply:
added in https://github.com/DataDog/dd-trace-py/pull/11730/commits/ec996b0fd2c2daa60e9f639d8c832dcd2339914f, you'd now need to either do `patch(langgraph=True)` or `DD_TRACE_LANGGRAPH_ENABLED=1` **along with** `_DD_TRACE_LANGGRAPH_ENABLED=1` in order for patching to happen.